pkg/channel/stats_reporter.go

```diff
@@ -54,13 +54,15 @@ var (
 	// - characters are printable US-ASCII
 	namespaceKey         = tag.MustNewKey(eventingmetrics.LabelNamespaceName)
 	eventTypeKey         = tag.MustNewKey(eventingmetrics.LabelEventType)
+	eventScheme          = tag.MustNewKey(eventingmetrics.LabelEventScheme)
 	responseCodeKey      = tag.MustNewKey(eventingmetrics.LabelResponseCode)
 	responseCodeClassKey = tag.MustNewKey(eventingmetrics.LabelResponseCodeClass)
 )
 
 type ReportArgs struct {
-	Ns        string
-	EventType string
+	Ns          string
+	EventType   string
+	EventScheme string
```

Comment:
In order for this new key to be reported anywhere, you will have to set it appropriately in `pkg/channel/fanout/fanout_event_handler.go` and `pkg/channel/event_receiver.go`, where the stats reporting for the channel is done